addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -59,7 +59,7 @@
                             </div>
                         </form>
                         <form class="d-none">
-                            <input type="file" class="o_portal_chatter_file_input" multiple="multiple"/>
+                            <input type="file" class="o_portal_chatter_file_input" multiple="multiple" onclick="this.value=null;"/>
```

Comment:
Feels like it could indeed be a solution but would not it appear "less hacky" to try to update that input value when an attachment is removed like in the described issue? My guess is that if I check the input value after an attachment removal, the value is inconsistent?

Reply:
Both values are currently unrelated: the file selector can be used several times without removing any attachment.
On every use, it will have the value of the last added attachment(s).
When popping the file selector, its start selection is not based on any of the specific previous attachments.
It behaves more like an "Add new attachment" than a "Update content of an attachment (if there was a previous one)".

The file selector value typically does not reflect a multiple selection upon reopening.
Imagine:
- attach "folder1/file1"
- attach "folder2/file2"
- open file selector again -> what should appear as the already selected state ?

Reply:
Ok I see, then your solution is the right one. However, it should not be in xml and from what I gather from what you say it should be less hacky than this: in the code which *adds* (and thus not *sets*) attachments post selection... empty the input file selection afterwards :+1:

Reply:
And maybe with a comment so that nobody removes that code thinking it is useless ;)